Simulator/player.py

```diff
@@ -178,6 +182,15 @@ def __init__(self, pool_pointer, player_num):
         self.default_agent = Default_Agent()
         self.default_player = False
 
+
+        # Store augments for the player
+        self.augments:List[SelectedAugmentConfig] = []
```

Comment:
This list is good. I just don't think we want to be calling the object that holds each Augment a config. A config is normally the parameters to run a large entity. We are holding the augments themselves rather than the parameters for a larger object? Although maybe I'm wrong on this one.